src/CSI.IBTA.Administrator/Clients/BenefitsServiceClient.cs

```diff
@@ -47,6 +46,44 @@ public async Task<GenericResponse<List<InsurancePackageDto>>> GetInsurancePackag
             var packages = JsonConvert.DeserializeObject<List<InsurancePackageDto>>(responseContent);
             return new GenericResponse<List<InsurancePackageDto>>(null, packages);
         }
+
+        public async Task<GenericResponse<InsurancePackageDto>> InitializeInsurancePackage(int packageId)
+        {
+            var token = _jwtTokenService.GetCachedToken();
```

Comment:
The project has class AuthorizedHttpClient, reuse it to avoid repeatable logic of token getting.